README.md

```diff
@@ -25,12 +26,67 @@ First you need to create a `vars.env` file with `PORT=3100` (Or any port you lik
+
+There will be a website where you can see all the data and all the explanation of the data. This is for the user who want's to look back and ponder on the amazing things the Ceuvel does.
+
+## Story
+What the Ceuvel does is making self made bio energy from plants and that energy will be needed to make some produce. That produce is used for the café.
+
+## User Stories/Stakeholders
+> As space&matter (a partner) I would want to know how the Ceuvel is doing on generating energy and how much it's sustainable energy is presented.
+
+> As gemeente Amsterdam (a partner) I would want to know why this place is good for the enviroment, and what they do better.
+
+## User scenario
+> Rick CTO of Space&matter wants to know more of the investment of the Ceuvel, and what the Ceuvel does.
+
+> Marit is a investment agent for the Gemeente Amsterdam. She wants to know what kind of benefit the Ceuvel has for Amsterdam and what the Ceuvel have researched and implemented.
+
+## (ideal) User flow
+Rick
+> .. When Rick arrive at the gate and/or done after having the introductory Rick would be seeing a sign to show me how to get started to see "more" insights in the Ceuvel. As Rick progresses he would see recognisable points to use so he can see more.
+
+
+Marit
+> ..Marit noticed that there are signs with stickers everywhere. She notice that with the application she will find more accomplishments the Ceuvel have achieved. With the interaction of these AR she has more insights. afterwards she can visit the website to see everything.
+
+## Goal
+1. The Goal of this project is to add an awareness of how bad grey energy is and show that the greenhouse in the Ceuvel a great alternative is.
+
+2. Learn about the Ceuvel and it's sustainability with green energy.
+
+3. The Ceuvel wants to let the partners know what they are doing and how they are doing it.
+
 # Features
-*The features will come here*
+* [d3.js](https://github.com/d3) - _d3JS is being used to render graphs with data_
+* [a-frame](https://aframe.io/) - _a frame is being used for the augmentend reality_
+* [SockJS](https://github.com/sockjs) - _SockJS is being used for the socket connection to the client_
+* [StompJS](https://www.npmjs.com/package/stompjs) - _StompJS is being used for the connection through the database to the server_
+* [ExpressJS](https://expressjs.com/) - _Express is being used for the server_
 
 # The data
-*Information about the data here*
 <!-- data life cycle? data retention/database ? -->
+The data is being stored on a stomp server and we use a connection via a StompJS package to get that data. What we receive is:
```

Comment:
The data is about the energy produces with solar energy and consumed. There is also a main array which we do not know the use of as of yet